src/main/java/org/janelia/saalfeldlab/paintera/viewer3d/OrthoSliceFX.java

```diff
@@ -73,6 +77,20 @@
 		});
 	}
 
+	private final DoubleProperty opacity = new SimpleDoubleProperty(1.0);
+	{
+		this.opacity.addListener((obs, oldv, newv) -> {
+			((PhongMaterial) this.meshView.get().getMaterial()).setDiffuseColor(new Color(
+					textureDiffuseOpaqueColor.getRed(),
```

Comment:
Could we just use [`Color.deriveColor`](https://docs.oracle.com/javafx/2/api/javafx/scene/paint/Color.html#deriveColor(double,%20double,%20double,%20double)) here?
```java
textureDiffuseOpaqueColor.deriveColor(0.0, 1.0, 1.0, newv.doubleValue());
```

Reply:
I didn't know about this method, I think I'll switch to it as you suggested to make it more concise